mne/viz/ica.py

```diff
@@ -60,6 +61,9 @@ def plot_ica_sources(ica, inst, picks=None, start=None,
     show_first_samp : bool
         If True, show time axis relative to the ``raw.first_samp``.
     %(show_scrollbars)s
+    time_format : 'float' | 'datetime
+        If 'datetime', the labels for x-axis ticks will be the time derived
+        from ``Info['meas_date']``. Default is 'float'.
```

Comment:
```suggestion
    time_format : 'float' | 'datetime'
        Style of time labels on the horizontal axis. If 'float', labels will be
        number of seconds from the start of the recording. If 'datetime',
        labels will show "clock time" (hours/minutes/seconds) inferred from
        ``raw.info['meas_date']``. Default is 'float'.
        
        .. versionadded:: 0.24
```

writing this makes me wonder if "datetime" is really the right parameter value (since we don't show dates).  Is "timestamp" better?  Or is "datetime" understood to mean "date and/or time in ISO 8601 format"?

Reply:
I agree that "datetime" is not optimal. What about "clocktime"? Or just "clock"?

Reply:
I like clock, because the ticks look like one and we never show a date.